components/AskAboutGroupModal.js

```diff
@@ -125,82 +84,88 @@ export default (props) => {
         </Subtitle>
-        <TextInput
-          autoCompleteType="name"
-          autoCorrect={false}
-          keyboardAppearance="dark"
-          returnKeyType="done"
-          placeholder="First name"
-          placeholderTextColor={Colors.gray}
-          value={firstName}
-          onChangeText={(value) => dispatch({ type: 'setFirstName', value })}
-          style={styles.input}
-        />
-        <TextInput
-          autoCompleteType="name"
-          autoCorrect={false}
-          keyboardAppearance="dark"
-          returnKeyType="done"
-          placeholder="Last name"
-          placeholderTextColor={Colors.gray}
-          value={lastName}
-          onChangeText={(value) => dispatch({ type: 'setLastName', value })}
-          style={styles.input}
-        />
-        <TextInput
-          autoCapitalize="none"
-          autoCompleteType="email"
-          keyboardAppearance="dark"
-          returnKeyType="done"
-          autoCorrect={false}
-          keyboardType="email-address"
-          placeholder="Email"
-          placeholderTextColor={Colors.gray}
-          value={email}
-          onChangeText={(value) => dispatch({ type: 'setEmail', value })}
-          style={styles.input}
-        />
-        <TextInput
-          keyboardAppearance="dark"
-          placeholder="Ask a question..."
-          placeholderTextColor={Colors.gray}
-          multiline={true}
-          value={question}
-          onChangeText={(value) => dispatch({ type: 'setQuestion', value })}
-          style={styles.input}
-        />
-        <View style={styles.submitButton}>
-          <Button title="Ask" onPress={handleAsk} />
-        </View>
-      </View>
 
-      <DropdownAlert
-        ref={dropdownAlertRef}
-        wrapperStyle={{ marginTop: Platform.OS === 'ios' ? 0 : 80 }}
-        renderImage={() => (
-          <AntDesign
-            name={'warning'}
-            size={30}
-            color={Colors.white}
-            style={{ padding: 8, alignSelf: 'center' }}
-          />
-        )}
-        zIndex={1}
-      />
+        <Formik
+          initialValues={{
+            firstName: '',
+            lastName: '',
+            email: '',
+            question: '',
+          }}
+          validate={validateAskQuestionForm}
+          onSubmit={handleAsk}
+        >
+          {({
+            handleChange,
+            handleBlur,
+            handleSubmit,
+            touched,
+            values,
+            errors,
+          }) => (
+            <View>
+              <Input
+                autoCompleteType="name"
+                label="First Name"
+                placeholder="Andy"
+                textContentType="givenName"
+                touched={touched.firstName}
+                value={values.firstName}
+                errors={errors.firstName}
+                onChangeText={handleChange('firstName')}
+                onBlur={handleBlur('firstName')}
```

Comment:
this stuff if kinda boilerplate, but i think abstracting it is unnecessary

Reply:
hmm, i can kinda think of a good way to abstract this...but i'll do it in a separate PR